### ISSUE TITLE
Initialize DataFrame without specifying order

### ISSUE THREAD

**Issue by lokeshh** [SciRuby/daru#130]
I was working with DataFrame initialization and think following would be convenient to have. What are your thougts?

``` ruby
# Initialization with array of vectors or arrays without specifying order.
>>> a = b = (1..3).dv
>>> Daru::DataFrame.new [a, b]
#<Daru::DataFrame:27152780 @name = 9e6a7851-96f6-43cb-93af-3ef59081e9a2 @size = 3>
                    0          1 
         0          1          1 
         1          2          2 
         2          3          3 

# Initialization with array of Enumerable like ranges without specifying order.
>>> Daru::DataFrame.new [1..3, 1..3]
#<Daru::DataFrame:26079900 @name = b0fb36c0-fd41-4a7e-9a02-36823cc9e1a5 @size = 3>
                    0          1 
         0          1          1 
         1          2          2 
         2          3          3 
```

**Comment by v0dro** [comment]
The first approach is alright (one with Vectors).

**Comment by athityakumar** [comment]
Fixed with PR #329 - I think this issue can be closed now. :smile:
